discussion/src/main/java/org/openedx/discussion/presentation/ui/DiscussionUI.kt

```diff
@@ -170,11 +168,14 @@ fun ThreadMainItem(
                     thread.voteCount,
                     thread.voteCount
                 ),
+                modifier = Modifier.alpha(if (thread.isAuthor) 0.4f else 1f),
```

Comment:
need the reference to use `0.4f` for disable state. we are using `0.3f` to present the disabled buttons.